tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -1302,28 +1305,38 @@ public function jobStatusValuesShouldMatchRegEx($user, $table) {
 	 * @param string $path
 	 * @param string $type
 	 *
-	 * @return ResponseInterface
+	 * @return void
 	 * @throws \Exception
 	 */
 	public function asFileOrFolderShouldNotExist(
-		$user, $entry, $path, $type = "files"
+		$user, $entry="file", $path= null, $type = "files"
 	) {
 		$user = $this->getActualUsername($user);
-		$path = $this->substituteInLineCodes($path, $user);
-		$response = WebDavHelper::makeDavRequest(
-			$this->getBaseUrl(), $this->getActualUsername($user),
-			$this->getPasswordForUser($user), 'GET', $path,
-			[], null, 2, $type
+		$path = $this->substituteInLineCodes($path);
+		$this->responseXmlObject = $this->listFolder(
+			$user, $path, 0, null, $type
 		);
-
-		if ($response->getStatusCode() < 401 || $response->getStatusCode() > 404) {
-			throw new \Exception(
-				"$entry '$path' expected to not exist " .
-				"(status code {$response->getStatusCode()}, expected 401 - 404)"
-			);
+		if (\gettype($this->responseXmlObject === "ResponseInterface")) {
```

Comment:
```suggestion
		if (\gettype($this->responseXmlObject) === "ResponseInterface") {
```
IMO the brackets need fixing

Reply:
fixed